docs/CONFIG-VARS.md

```diff
@@ -255,10 +261,10 @@ V4_CFG_POSTGRES_SERVERS:
 | backrest_pvc_storage_size | Size of the internal pgBackrest PVCs | string | 128Gi | false | This value can be changed but not decreased after the initial deployment. Supported for cadence versions 2022.10 and later. Only for internal databases.| viya |
 | postgres_pvc_access_mode | Access mode for the PostgreSQL PVCs | string | ReadWriteOnce | false | Supported values: [`ReadWriteOnce`,`ReadWriteMany`]. This value cannot be changed after the initial deployment. Supported for cadence versions 2022.10 and later. Only for internal databases.| viya |
 | backrest_pvc_access_mode | Access mode for the pgBackrest PVCs | string | ReadWriteOnce | false | Supported values: [`ReadWriteOnce`,`ReadWriteMany`]. This value cannot be changed after the initial deployment. Supported for cadence versions 2022.10 and later. Only for internal databases.| viya |
-| postgres_storage_class | Storage class for the PostgreSQL PVCs | string | sas | false |This value cannot be changed after the initial deployment. Supported for cadence versions 2022.10 and later. Only for internal databases.| viya |
-| backrest_storage_class | Storage class for the pgBackrest PVCs | string | sas | false |This value cannot be changed after the initial deployment. Supported for cadence versions 2022.10 and later. Only for internal databases.| viya |
+| postgres_storage_class | Storage class for the PostgreSQL PVCs | string | pg-storage | false |This value cannot be changed after the initial deployment. Supported for cadence versions 2022.10 and later. Only for internal databases. The default value is only used if `V4_CFG_MANAGE_STORAGE` is set to `true`. If `V4_CFG_MANAGE_STORAGE` is set to false, set the value to your pre-existing StorageClass name. | viya |
+| backrest_storage_class | Storage class for the pgBackrest PVCs | string | pg-storage | false |This value cannot be changed after the initial deployment. Supported for cadence versions 2022.10 and later. Only for internal databases. The default value is only used if `V4_CFG_MANAGE_STORAGE` is set to `true`. If `V4_CFG_MANAGE_STORAGE` is set to false, set the value to your pre-existing StorageClass name. | viya |
```

Comment:
Same as above.

Reply:
Was this one not changed or was it reviewed?

Reply:
@thpang I don't see the suggestion that you just resolved for line 264 in the commits. If you didn't also apply the suggested change for line 264 I will update 264 to match the same suggestion for line 265 below.

Reply:
@thpang lines 264 & 265 have now been updated with Susan's suggested text, please comment or resolve this one if you are good with that result.